Assets/Countly/Plugins/CountlySDK/Interfaces/IViewModule.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+// interface for SDK users

```

Comment:
What happened to the other view calls? resume, pause, stop etc

Reply:
Since that PR is too big we are dividing it into smaller ones and while doing that, it should not break anything, so this PR introduces interface without new methods, after we merge this one, a new PR will introduce the view calls, which is for everyone to review much easily and detect any errors if there are